docs/rfcs/0000-template.md

```diff
@@ -0,0 +1,82 @@
+- Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
+
+# Summary
+[summary]: #summary
```

Comment:
```suggestion
```

Reply:
I don't think these hyperlink tags are useful enough to be worth the confusion they can cause beginners / visual clutter. As far as I can tell, you can already link to mdbook sections just fine.